wgengine/router/ifconfig_windows.go

```diff
@@ -387,3 +379,25 @@ func configureInterface(cfg *Config, tun *tun.NativeTun) error {
 
 	return errAcc
 }
+
+// routeLess reports whether ri should sort before rj.
+// The actual sort order doesn't appear to matter. The caller just
+// wants them sorted to be able to de-dup.
+func routeLess(ri, rj *winipcfg.RouteData) bool {
+	if ri.Metric != rj.Metric {
```

Comment:
As a minor aesthetic preference, I normally prefer to keep routes together that are identical except for metric. The original ordering (other than bugs) should match the order Linux sorts its routing tables, and would result in "first match is best match" semantics as normally expected in a routing table.

We might want to keep that ordering just in case someone tries to use this function for something more semantically sensitive later.

Reply:
Done, with new tests.